.github/workflows/build.yaml

```diff
@@ -28,7 +28,7 @@ concurrency:
 jobs:
   cpp-build:
     secrets: inherit
-    uses: rapidsai/shared-action-workflows/.github/workflows/conda-cpp-matrix-build.yaml@main
+    uses: rapidsai/shared-action-workflows/.github/workflows/conda-cpp-build.yaml@cuda-118
```

Comment:
just curious - do these eventually go back to being `main`, or is this a new naming convention moving forward?

Reply:
Yes, @ajschmidt8 has a plan for this. I am not yet sure what it will look like.

Reply:
> Seems okay to me.
> 
> One request though: can you briefly summarize what, if any, steps cugraph devs will need to do once this is merged? We're approaching burndown and don't want to get derailed with any surprises.

No further steps are necessary. Once this PR is merged, a build will kick off to publish the latest `11.8` / `3.10` packages.

It's recommended that any outstanding PRs merge in the latest changes so that they can also test against `11.8` and `3.10`. But only actively developed PRs, not a bulk wholesale update since that would overload CI.


> just curious - do these eventually go back to being main, or is this a new naming convention moving forward?

Yes, once all the repos are using the `cuda-118` branch to build against `11.8` and publish `3.10` packages, we'll merge that branch into `main` and open up subsequent PRs across all the repositories to update this branch again.

I think we're just waiting for this `cuml` PR at this point: https://github.com/rapidsai/cuml/pull/5120